cetmix_tower_server/models/cx_tower_server_template.py

```diff
@@ -171,6 +171,10 @@ def create_server_from_template(self, template_reference, server_name, **kwargs)
                     `variable_reference`: `variable_value_char`
                     eg:
                     {'branch': 'prod', 'odoo_version': '16.0'}
+            with_template_variables (bool):  This parameter ensures that the server
```

Comment:
Could you please elaborate this behaviour?
Suppose we have a Server Template with variables "A", "B", "C" and "D".
If I will run this function with  `with_template_variables = True` variables "A", "C" and "R",  what exactly happens?

Reply:
@ivs-cetmix 

If with_template_variables = True, the server will be created with variables "A", "B", "C", "D", and "R". This is useful when the server creation method is called from a server action, as it ensures all template variables are included along with the specified ones.

However, if with_template_variables = False, only the specified variables "A", "C", and "R" will be considered. This is the default behavior when creating a server from a wizard, where the wizard lines explicitly specify the required variables from the template without fully copying all template variables.